pyramid_okta/security/authentication.py

```diff
@@ -65,6 +64,7 @@ def effective_principals(self, request):
             effective_principals.append(user_id)
         else:
             effective_principals.append(Authenticated)
+            effective_principals.append('login_required')
```

Comment:
is that needed?  I think `'login_required'` is the permission, and `Authenticated` is a principal that will return `True` for that permission.